<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_tranaccounts</class>
 <widget class="QDialog" name="user_tranaccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/log1.jpg);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请输入密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="user_passward1">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDialogButtonBox" name="yesorno">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="close_music">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>关闭音乐</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请输入用户账号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请输入身份证号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="user_id">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="user_num">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请确认密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="user_passward2">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请输入转账金额</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="tran_money">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>请输入接收人账号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="tran_people">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
